### PR TITLE
Feature points represented with dots

### PR DESCRIPTION
**Pull request by Veichassur**
Quite a hacky workaround by identifing modified feature values by id to update them in actor.items.
I just really wanted to see dots instead of pure numbers for the features section.

### CODE REVIEW

**Comment by Veichassur** [comment]
image

**Comment by Rayji96** [comment]
Can you make it a read-only display of dots?
It breaks the workflow of items only being editable in the item sheet.

**Comment by Veichassur** [comment]
Hmm, that' true.. tbh I don't like that workflow very much though. I just
handled the creation of only a handful of characters but could already feel
that it's rather tiresome to always have to open the item sheet just to
distribute points.
I would even argue that the point number is inherently part of the
character, not the item itself.
From a UX standpoint the workflow should be: pick a merit/flaw/background
from a compendium and drag&drop that onto the character sheet. Then
distribute points for that item per-character.

Maybe go the other way and make other fields also editable directly on the
character sheet?
What do you think?

Oh one more thought: as a user it should feel similar to the disciplines
list I think.

**Comment by Rayji96** [comment]
The points never change for a merit or flaw, do they?
I also agree the workflow is a bit slow if you don't have a merit compendium already but I also do want to avoid too much coupling.

**Comment by Veichassur** [comment]
Yeah, merits are flaws can be handled without too much hassle in the long run, I was mainly thinking about background points as features, those can change quite frequently, even during campaign.
So, how about a static read-only dot display for merits/flaws but changeable dots for backgrounds?

**Comment by Veichassur** [comment]
Looks like this now, merits/flaws use a different div class so they're not modifiable.

image

File: module/actor/actor-sheet.js [hunk 1]
```diff
@@ -332,7 +332,7 @@ export class VampireActorSheet extends ActorSheet {
       data.skill = ''
     }
     if (type === 'boon') {
-      data.boontype = "Trivial"
+      data.boontype = 'Trivial'
```
Comment (Rayji96) [inline module/actor/actor-sheet.js:335]: Should probably use a default value that draws from localization but I'm cool without :)

File: module/actor/actor-sheet.js [hunk 2]
```diff
@@ -592,8 +592,19 @@ export class VampireActorSheet extends ActorSheet {
...
-    fields.reduce((data, field) => data[field], actorData)[lastField] = value
+    // update actor owned items
+    if (fields.length === 2 && fields[0] === "items") {
+      for (const i of actorData.items) {
+        if (fields[1] === i._id) {
+          i.data.points = value
+          break
+        }
+      }
+    }
```
Comment (Rayji96) [inline module/actor/actor-sheet.js:603]: Should use a reduce here. I'll try to figure it out. :)

**Review by Rayji96 — APPROVED** [review]

**Pull request merged**